flight_termination/src/scripts/flight_termination_controller.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+import sys
+
+# Add parent directory to Python path at runtime to allow importing packages.
+sys.path.append("../")
```

Comment:
Hi @nakulsodhi, do you know a better way to do this? I had issues importing packages from `flight_termination` in this script. This appears to be sort of standard though.

Reply:
This is a pretty valid way of doing this. I personally couldn't find anything better than this.